apps/emqx_conf/src/emqx_cluster_rpc.erl

```diff
@@ -527,10 +533,13 @@ wait_for_nodes_commit(RequiredSyncs, TnxId, Delay, Remain) ->
         false when Remain > 0 ->
             wait_for_nodes_commit(RequiredSyncs, TnxId, Delay, Remain - Delay);
         false ->
-            case lagging_node(TnxId) of
```

Comment:
it's old code, but shouldn't it be `lagging_nodes` instaed?

Reply:
good catch!